pom.xml

```diff
@@ -16,8 +16,28 @@
 		<underscorejs.version>1.8.3</underscorejs.version>
 	</properties>
 
-	<dependencies>
 
+	<dependencies>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-core</artifactId>
+			<version>2.4.3</version>
+		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+			<version>2.4.3</version>
```

Comment:
Without this dependency it shows this message: No converter found for return value of type: class com.iri.training.model.User